runtime/cpp/instance_segmentation/yolov5seg/README.md

```diff
@@ -19,15 +19,20 @@ Prints the object detected + confidence to the screen
 Prints statistics
 NOTE: Currently supports only devices connected on a PCIe link.
 
-Prequisites: OpenCV 4.2.X CMake >= 3.20 HailoRT >= 4.10.0 git - rapidjson repository is cloned when performing build.
-
-To compile the example run `./build.sh`
-
-To run the compiled example:
-
-
-`./build/vstream_yolov5seg_example_cpp -hef=yolov5n_seg.hef -input=full_mov_slow.mp4`
-
-NOTE: You can also save the processed video by commenting in a few lines at the `post_processing_all` function in yolov5seg_example.cpp.
+1. Dependencies:
+    - OpenCV, and g++-9:
+    ``` bash
+    sudo apt-get install -y libopencv-dev gcc-9 g++-9
+    ```
+2. Build the project build.sh
+3. Run the executable:
+    ``` bash
+    ./build/segmentation_example_cpp -hef=HEF_PATH -path=VIDEO_PATH
```

Comment:
Not necessarily a video, so change to "input_path". Better to have it "-input" rather than "-path" as it is more indicative.